src/includes/ConvertImageToByte.js

```diff
@@ -1,14 +1,19 @@
 const reader = new FileReader();
 const fileByteArray = [];
 
-function getConvertedImage(data) {
+/**
+ * Converts the image file to byte array.
+ * @param {file} image file.
```

Comment:
does it support image files with only some specific extensions like .jpg?

Reply:
yes it supports